addons/default/visiosoft/profile-module/src/Http/Controller/MyProfileController.php

```diff
@@ -246,4 +249,24 @@ public function updateAjaxProfile(UserRepositoryInterface $user)
         return response()->json(['status' => 'success', 'data' => $profile]);
     }
 
+	public function getEducation(Request $request)
+	{
+		$user = $this->userRepository->find(auth()->user()->getAuthIdentifier());
+		$education = EducationModel::all();
+		$educationPart = EducationPartModel::query()->where('education_id', $user->education)->get();
+		return response()->json(['user' => $user, 'education' => $education, 'education-part' => $educationPart], 200);
+	}
+
+	public function setEducation(Request $request)
+	{
+		$user_id = auth()->user()->getAuthIdentifier();
```

Comment:
why not just use auth()->user()?

Reply:
just need id

![resim](https://user-images.githubusercontent.com/22885454/102230818-5d212280-3efe-11eb-849b-056ad329dc20.png)